experimenter/experimenter/experiments/tests/api/v5/test_queries.py

```diff
@@ -2298,9 +2302,22 @@ def test_query_excludes_other_applications(self):
             )
 
 
+@override_settings(
+    CACHES={
+        "default": {
+            "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+        },
+    },
+)
 class TestNimbusConfigQuery(GraphQLTestCase):
     GRAPHQL_URL = reverse("nimbus-api-graphql")
 
+    def setUp(self):
+        super().setUp()
+        cache.delete(SIZING_DATA_KEY)
+        sizing_test_data = SampleSizesFactory.build()
+        cache.set(SIZING_DATA_KEY, sizing_test_data)
```

Comment:
Wait if we're just setting the same data (random here but looks indistinguishable from the test perspective?) then shouldn't we only need to do this once, ie in `setUpClass`?

Reply:
And then if you have to do the same override and setup in multiple test cases, I'd pull it out into a single mixin class and just mix that in so you don't have to repeat it verbatim every time?